src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -4,15 +4,22 @@ import { NavigationMixin } from 'lightning/navigation';
 import messageLoading from '@salesforce/label/c.labelMessageLoading';
 import geSave from '@salesforce/label/c.labelGeSave';
 import geCancel from '@salesforce/label/c.labelGeCancel';
+import geDonationTypeErrorLabel from '@salesforce/label/c.geErrorDonorTypeValidation';
 import geUpdate from '@salesforce/label/c.labelGeUpdate';
 import { showToast, handleError, getRecordFieldNames, setRecordValuesOnTemplate } from 'c/utilTemplateBuilder';
 import { getQueryParameters, isNotEmpty } from 'c/utilCommon';
 import { getRecord } from 'lightning/uiRecordApi';
+import { format, isEmpty } from 'c/utilCommon';
```

Comment:
Can we consolidate this line with line 10?

Reply:
sure will do